routes/cartRouter.js

```diff
@@ -1,11 +1,14 @@
 const router = require('express').Router()
-const {
-  addCart
-} = require('../controllers/cart')
+const { addCart, getCart } = require('../controllers/cart')
 
 // @route POST api/store/:store_id/cart
 // @desc  Add items to cart
 // @access Public
 router.post('/:store_id/cart', addCart)
 
+// @route GET api/store/:store_id/cart
+// @desc  Get items from cart
+// @access Public
+router.get('/:cart_id/', getCart)
```

Comment:
I think this route is a bit confusing /api/store/123(id)
Even if you are not using the store id, you can go with /api/store/cart/123(id)